core/src/main/scala/higherkindness/mu/rpc/srcgen/FileUtil.scala

```diff
@@ -0,0 +1,19 @@
+package higherkindness.mu.rpc.srcgen
+
+import java.io._
+
+object FileUtil {
+
+  implicit class FileOps(val file: File) extends AnyVal {
+
+    def write(lines: Seq[String]): Unit = {
```

Comment:
I had to copy these methods from mu-scala. They are used by the Avro code generators.